chart/values.yaml

```diff
@@ -195,6 +200,16 @@ cacheRefresh:
       cpu: 0
   tolerations: []
 
+cacheMaintenance:
+  # action: "backfill" -- Set this value according to required action for maintenance
```

Comment:
maybe set the default action here (since the job will use one action anyway, I think it's better to be explicit here about what is the default action)

Reply:
Thanks, I fixed it on https://github.com/huggingface/datasets-server/pull/1017/commits/e67005f028c2117e44c80a88f7d170ab664afaef